src/main/java/fi/aalto/cs/apluscourses/presentation/MainViewModel.java

```diff
@@ -22,4 +22,14 @@
           courseViewModel,
           authenticationViewModel,
           ExercisesTreeViewModel::fromCourseAndAuthentication);
+
+  @NotNull
+  public ObservableProperty<APlusAuthenticationViewModel> getAuthenticationViewModel() {
```

Comment:
getting issues when trying to mock the object references (with `.` operator), so that's why getters :confused: